src/pywink/test/init_test.py

```diff
@@ -45,771 +55,256 @@ def test_should_only_send_color_xy_if_both_color_xy_and_color_temperature_are_gi
-                    "position": 0,
-                    "position_updated_at": 1450357537.836,
-                    "battery": null,
-                    "battery_updated_at": null,
-                    "fault": false,
-                    "fault_updated_at": 1447976866.0784,
-                    "disabled": null,
-                    "disabled_updated_at": null,
-                    "control_enabled": true,
-                    "control_enabled_updated_at": 1447976866.0784,
-                    "desired_position_updated_at": 1447976846.8869,
-                    "connection_changed_at": 1444775470.5484,
-                    "position_changed_at": 1450357537.836,
-                    "control_enabled_changed_at": 1444775472.2474,
-                    "fault_changed_at": 1444775472.2474,
-                    "position_opened_changed_at": 1450357467.371,
-                    "desired_position_changed_at": 1447976846.8869
-                },
-                "garage_door_id": "30528",
-                "name": "Garage Door",
-                "locale": "en_us",
-                "units": {
-
-                },
-                "created_at": 1444775470,
-                "hidden_at": null,
-                "capabilities": {
-                    "home_security_device": true
-                },
-                "triggers": [
-
-                ],
-                "manufacturer_device_model": "chamberlain_garage_door_opener",
-                "manufacturer_device_id": "1133930",
-                "device_manufacturer": "chamberlain",
-                "model_name": "MyQ Garage Door Controller",
-                "upc_id": "26",
-                "upc_code": "012381109302",
-                "hub_id": null,
-                "local_id": null,
-                "radio_type": null,
-                "linked_service_id": "206203",
-                "lat_lng": [
-                    0,
-                    0
-                ],
-                "location": "",
-                "order": null
-            }],
-            "errors": [],
-            "pagination": {}
-        }
-        """
-        response_dict = json.loads(response)
+        with open('{}/api_responses/garage_door.json'.format(os.path.dirname(__file__))) as garage_door_file:
+            response_dict = json.load(garage_door_file)
         devices = get_devices_from_response_dict(response_dict, DEVICE_ID_KEYS[device_types.GARAGE_DOOR])
         self.assertEqual(1, len(devices))
         self.assertIsInstance(devices[0], WinkGarageDoor)
+        
+    def test_device_id_should_be_number(self):
+        with open('{}/api_responses/garage_door.json'.format(os.path.dirname(__file__))) as garage_door_file:
+            response_dict = json.load(garage_door_file)
+        garage_door = response_dict.get('data')[0]
+        wink_garage_door = WinkGarageDoor(garage_door, self.api_interface)
+        device_id = wink_garage_door.device_id()
+        self.assertRegex(device_id, "^[0-9]{4,6}$")
```

Comment:
This should be only a number ^ = beginning of line and $ = end of line. Now on the sensor I am only checking that it starts with a number since it also returns the capability.